src/main/java/lotto/domain/BonusNum.java

```diff
@@ -0,0 +1,32 @@
+package lotto.domain;
+
+public class BonusNum {
+
+    private static final int MIN_ELEMENT = 1;
+    private static final int MAX_ELEMENT = 45;
+    private int value;
+
+    public BonusNum(int bonusNum, Lotto lotto){
+        checkBound(bonusNum);
+        for (int input: lotto.getLottoList()) {
+            validBonusNum(input, bonusNum);
+        }
+            this.value = bonusNum;
+    }
+
+    public void validBonusNum(int input, int bonusNum){
```

Comment:
```suggestion
    private void validBonusNum(int input, int bonusNum){
```

여기는 지난 번에 놓친 부분이네요.
외부로 노출하지 않을 메소드는 private로 변경해주세요.